assets/persistence/example_data.gd

```diff
@@ -0,0 +1,22 @@
+extends PersistentDataSection
+
+const PD_SECTION_EXAMPLE = "example"
+const PD_SECTION_EXAMPLE_TIME = "time"
+const PD_SECTION_EXAMPLE_VALUE = "value"
```

Comment:
@BenKellySoftware I got an email about a comment here, but it's not showing for me:
> Could we make these field name definitions an enum, or array that we can do the below logic as a map instead of copy pasting for each field?
So the answer is maybe? I had a quick think before about alternatives, but I couldn't think of anything that made sure both the serialise and deserialise functions would use the same field IDs without importing the whole blob unchecked. Do you or @Raelr have any ideas on this?